go-controller/pkg/clustermanager/node/node_allocator.go

```diff
@@ -149,7 +149,7 @@ func (na *NodeAllocator) releaseHybridOverlayNodeSubnet(nodeName string) {
 
 // HandleAddUpdateNodeEvent handles the add or update node event
 func (na *NodeAllocator) HandleAddUpdateNodeEvent(node *corev1.Node) error {
-	defer na.recordSubnetCount()
```

Comment:
now wont recordSubnetCount not happen during node event? So those metrics wont be updated?

Reply:
recordSubnetCount updates ovnkube_clustermanager_allocated_v4_host_subnets metric that tracks the number of subnets possible per network metric which is gonna be a constant value and we are updating this metric during the init function so no need to update this during node update/add or del event as its gonna be constant value

Reply:
ah OK. I thought it would be the number of used subnets only. Makes sense.